### PR TITLE
[MNT-24119] Update Supported Platforms for ActiveMQ CVE-2023-46604

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

**Review by Prosune — COMMENTED** [review]
Did not find changes in ATS and DTE, rest looks fine.

**Comment by anxumalo** [comment]
> Did not find changes in ATS and DTE, rest looks fine.

@Prosune The ATS changes were completed as part of PR #1290. See the previous comments about DTE - no updates needed.

**Review by Prosune — APPROVED** [review]
LGTM 👍

File: content-services/7.4/support/index.md
Comment (isandhu) [inline content-services/7.4/support/index.md:1]: Do we need to add Sync Service 3.11.1 here seeing that we already have 3.11 listed against ACS 7.4.1? For ACS 7.4.0 we do need to add it as 3.11 is NOT listed.

I don't mind if you wish to add it for consistency across all versions, just pointing it out. Thanks.

**Review by isandhu — APPROVED** [review]
LGTM

File: content-services/7.4/support/index.md
Comment (anxumalo) [reply content-services/7.4/support/index.md:1]: @isandhu Thanks - I'm adding it for consistency

**Pull request merged**